.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules
 .env
 android
 ios
+.history/
+pnpm-lock.yaml
```

Comment:
No need to ignore pnpm-lock.yaml we just shouldn't have one anymore now that we use bun